app/extensions.js

```diff
@@ -132,7 +132,6 @@ let generateBraveManifest = () => {
 }
 
 let defaultExtensions = {
-  PDFJS: 'oemmndcbldboiebfnladdacbdfmadadm',
```

Comment:
this ID is hard-coded in pdfjs's telemetry.js module, but luckily we don't use that at all